### ISSUE TITLE
Connection refused when using EventHubsTransportType.AmqpWebSockets

### ISSUE THREAD

**Issue by jdiekhoff-copeland** [Azure/azure-event-hubs-emulator-installer#32]
**Describe the bug**
Running containers in the same docker bridge network and attempting to connect via using EventHubsTransportType.AmqpWebSockets results in a connection refused.

**To Reproduce**

- Run the event hub container using tag `1.2.4-preview-amd64` as that [supports connecting to the event hub container via container alias](https://github.com/Azure/azure-event-hubs-emulator-installer/issues/11#issuecomment-2199432626). 
- Run a container that is pushing messages to the eventhub with Azure.Messaging.EventHub version 5.1.3 ([Known fix to support connecting without localhost restriction](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Messaging.EventHubs_5.11.3/sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md#5113-2024-05-15)) with TransportType set as AmqpWebSockets. 

Sample compose:

```yml
services:
  my-service:
    image: "my-image:latest"
    environment:
      EventHub__ConnectionString: Endpoint=sb://eventhub;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;
    networks:
      - network
    depends_on:
      - eventhub
  eventhub:
    image: "mcr.microsoft.com/azure-messaging/eventhubs-emulator:1.2.4-preview-amd64"
    volumes:
      - "./config.json:/Eventhubs_Emulator/ConfigFiles/Config.json"
    environment:
      BLOB_SERVER: eventhub-azurite
      METADATA_SERVER: eventhub-azurite
      ACCEPT_EULA: ${EH_ACCEPT_EULA}
    depends_on:
      - eventhub-azurite
    networks:
      - network
  eventhub-azurite:
    image: "mcr.microsoft.com/azure-storage/azurite:latest"
    networks:
      - network

networks:
  network:
```

**Expected behavior**
Container should be able to connect to the event hub.

**Screenshots**
![2024-08-21_20-23-31](https://github.com/user-attachments/assets/21b1ab63-8eeb-4494-8efe-44e7e3389b9b)

**Logs**
[log2024082200.txt](https://github.com/user-attachments/files/16701147/log2024082200.txt)

**Desktop:**
 - OS: Windows 10 Enterprise

**Image Platform-Architecture Used:**
- Linux-amd64

**Docker Version:**
- Docker Desktop Version (If applicable): 4.32.0
- Docker Engine Version: 27.0.3

**Arguments && Environment variables to start Emulator:**
- ACCEPT_EULA: Y
- CONFIG_PATH: "./config.json:/Eventhubs_Emulator/ConfigFiles/Config.json"

**Emulator Launch Method:**
 - Direct Docker compose

**Additional context:**

I am able to connect fine if I leave the TransportType to the default, but I don't see anything in the documentation for the emulator that AmqpWebSockets is unsupported.

**Comment by Saglodha** [comment]
Hi @jdiekhoff-copeland , thanks for bringing this to our attention. As of today, Event Hubs emulator only works with Default AMQP flow and AMQPWebsockets is not supported. We'll update our documentation to reflect this accordingly. 

To understand the situation better, could you please share more details on your use case and why you'd prefer AMQPwebsockets over traditional AMQP?

**Comment by jdiekhoff-copeland** [comment]
@Saglodha good to know, I will update our code accordingly. 

The use of AMQPwebsockets was being done for local development as a workaround. Prior to using the eventhub emulator, our development team was testing against Azure Eventhubs. We noticed when we were on VPN that the normal connection to EventHubs was not working, and we were getting socketexception 10054. We found this [stackoverflow](https://stackoverflow.com/questions/64558111/azure-eventhub-socketexception-10054-when-creating-createbatchasync-and-connecte) and by setting the TransportType to AMQPWebSockets allowed for normal operations.

Now that we are using the emulator, we do not have a use case for AMQPWebSockets since all traffic is local.
